src/transformers/generation/flax_utils.py

```diff
@@ -377,11 +377,11 @@ def generate(
         # Prepare `max_length` depending on other stopping criteria.
         input_ids_seq_length = input_ids.shape[-1]
         has_default_max_length = kwargs.get("max_length") is None and generation_config.max_length is not None
-        if has_default_max_length and generation_config.max_new_tokens is None:
+        if has_default_max_length and generation_config.max_new_tokens is None and generation_config.max_length != 20:
+            # 20 is the default max_length of the generation config
```

Comment:
Can we pull that value dynamically? Or add a comment next to the default of the config to also update all those tests in case of change in the future?

Reply:
Can't pull it dynamically but will add a comment!